exporters/css/src/index.ts

```diff
@@ -34,16 +34,19 @@ Pulsar.export(async (sdk: Supernova, context: PulsarContext): Promise<Array<AnyO
     tokenGroups = tokenGroups.filter((tokenGroup) => tokenGroup.brandId === brand.id)
   }
 
-  // Apply theme, if specified
-  if (context.themeId) {
+  // Apply themes, if specified
+  if (context.themeIds && context.themeIds.length > 0) {
     const themes = await sdk.tokens.getTokenThemes(remoteVersionIdentifier)
-    const theme = themes.find((theme) => theme.id === context.themeId || theme.idInVersion === context.themeId)
-    if (theme) {
-      tokens = await sdk.tokens.computeTokensByApplyingThemes(tokens, [theme])
-    } else {
-      // Don't allow applying theme which doesn't exist in the system
-      throw new Error("Unable to apply theme which doesn't exist in the system.")
-    }
+
+    const themesToApply = context.themeIds.reduce((acc, themeId) => {
+      const theme = themes.find((theme) => theme.id === themeId || theme.idInVersion === themeId)
+      if (theme) {
+        acc.push(theme)
+      }
```

Comment:
Should we throw error in case theme won't be found?

Reply:
But based on thrown error on not found brand we should: 1e24ef8dff90402c237d7220f0f3029acdfc0863